draft-ietf-acme-acme.md

```diff
@@ -1699,13 +1763,25 @@ Content-Type: application/jose+json
 
 The CSR encodes the client's requests with regard to the content of the
 certificate to be issued.  The CSR MUST indicate the exact same set of requested
-identifiers as the initial new-order request, either in the commonName portion
+identifiers as the initial new-order request.  Identifiers of type "dns" MUST apper in either in the commonName portion
```

Comment:
typo: "MUST apper" -> "appear"

typo 2: "MUST apper in either in the" -> "MUST appear in either the"